charts/microgateway/README.md.gotmpl

```diff
@@ -322,17 +322,23 @@ The use cases outlined above can also occur slightly differently. But all of the
                       pattern: ^/mail/
                     method:
                       pattern: ^POST$
+              backend:
+                hosts:
+                  protocol: https
```

Comment:
back-end hosts is a list entry, for that reason a hyphen character is required.

```
hosts:
- protocol: https
```

Reply:
fixed